src/safeds_runner/server/_memoization_map.py

```diff
@@ -290,21 +295,47 @@ def _update_stats_on_miss(
         self._map_stats[function_name] = stats
 
 
-def _convert_list_to_tuple(values: list) -> tuple:
+def _make_serializable(value: Any) -> Any:
```

Comment:
I don't fully agree on the name here, as most of the types are made `hashable` by calling this function, while already being serializable.

Reply:
Would `_make_hashable` make sense then?

Reply:
As it is only used during the creation of the memoization key, this would better convey what this function is doing, in my opinion :+1: